presto-orc/src/test/java/com/facebook/presto/orc/TestDecryption.java

```diff
@@ -208,22 +209,22 @@ public void testGetStripeDecryptionKeysUnencrypted()
     public void testGetDiskRanges()
     {
         List<Stream> unencryptedStreams = ImmutableList.of(
-                new Stream(3, ROW_INDEX, 5, true, DEFAULT_SEQUENCE_ID, Optional.of(15L)),
-                new Stream(4, ROW_INDEX, 5, true),
-                new Stream(3, DATA, 5, true, DEFAULT_SEQUENCE_ID, Optional.of(45L)),
-                new Stream(4, DATA, 5, true));
+                new Stream(3, DEFAULT_SEQUENCE_ID, ROW_INDEX, 5, true, Optional.of(15L)),
+                new Stream(4, ColumnEncoding.DEFAULT_SEQUENCE_ID, ROW_INDEX, 5, true),
```

Comment:
static import DEFAULT_SEQUENCE_ID instead of using the fully qualified ColumnEncoding.DEFAULT_SEQUENCE_ID (here and throughout this PR)